bin/scripts/fix-data/recheck_merges.js

```diff
@@ -0,0 +1,89 @@
+/**
+ *  Description: This script is used to recheck merges and update drill data with new uid
+ *  Server: countly
+ *  Path: $(countly dir)/bin/scripts/fix-data
+ *  Command: node recheck_merges.js
+ */
+
+
+var asyncjs = require("async");
+const pluginManager = require('../../../plugins/pluginManager.js');
+const dataviews = require('../../../plugins/drill/api/parts/data/dataviews.js');
+
+Promise.all([pluginManager.dbConnection("countly"), pluginManager.dbConnection("countly_drill")]).then(async function([countlyDb, drillDb]) {
+    console.log("Connected to databases.");
+    //get all apps
+    countlyDb.collection("apps").find().toArray(function(err, apps) {
+        if (err) {
+            return close(err);
+        }
+        //get all drill collections
+        drillDb.collections(function(err, collections) {
+            if (err) {
+                return close(err);
+            }
+            //for each app
+            asyncjs.eachOf(apps, function(app, index, done) {
+                var app_id = app._id;
+                if (err) {
+                    return done();
+                }
+                //get all users with merges
+                countlyDb.collection('app_users' + app_id).find({merges: {$gt: 0}}).toArray(function(err, users) {
```

Comment:
Do not use to array. Use cursor. If they have 10 million users this will read them all in memory